test/e2e/network/dns.go

```diff
@@ -21,7 +21,7 @@ import (
 	"strings"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Seems like an unnecessary change.

Reply:
Removed, my editor did this behind my back ...